xbmc/music/tags/MusicInfoTag.h

```diff
@@ -204,6 +212,8 @@ class CMusicInfoTag final : public IArchivable, public ISerializable, public ISo
   std::string m_strLyrics;
   std::string m_cuesheet;
   std::string m_strDiscSubtitle;
+  std::string m_strOrigYear;
+  std::string m_strMonth;
```

Comment:
Hum..... having month in CMusicInfoTag is bugging me. The info tag stuff is messy enough historically would be good to avoid making it worse. I'm not happy having this just for TDAT from FFMpeg, so can't this be avoided by using the date field intelligently.

Also declare m_strAlbumReleaseDate and m_strOrigYear together

Reply:
I don't think it can.  We don't know the order of field reading so have to store the month somewhere in case it's read first.  At that point we cannot use it in a date because we don't know the year (or even if there is going to be one).  If we put TDAT into the date field, expecting to get a year to put in front of it, and then there is no year, the rest of the processing will consider something like '0515' as the year.  I guess we could use 1900 or something as a default year ?